README.md

```diff
@@ -41,5 +41,5 @@ Syntax:
 |[Nonso Andrew](https://twitter.com/nonso_andy)| Happy Birthday, Eddie!|
 |[Aditya Painuli](https://twitter.com/aditya_painuli)| Happy birthday open source hero wish you a good health any looking forward for more contrubtion🙌🎉🎉|
 |[Shivendu Kumar](https://twitter.com/shivendu_kr_jha)| Many Many Happy Returns of the Day Eddie 🥳🎉! Wish you have a great health & career ahead!!✨🧡💓|
-|[Chandraprakash Darji](http://twitter.com/chandra_7852)| Happy Birthday, Eddie! You are the inspiration for Open Source, Remote Work, and freelancing. Thanks. |
+|[Aayush Sharma](https://twitter.com/SuperAayush)| Happy Birthday to the person from whom I learned many good Open-Source practices, May you have a long and happy life ❤️🍾🎂|
```

Comment:
```suggestion
|[Chandraprakash Darji](http://twitter.com/chandra_7852)| Happy Birthday, Eddie! You are the inspiration for Open Source, Remote Work, and freelancing. Thanks. |
|[Aayush Sharma](https://twitter.com/SuperAayush)| Happy Birthday to the person from whom I learned many good Open-Source practices, May you have a long and happy life ❤️🍾🎂|
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix